lib/api/element-commands/click.js

```diff
@@ -51,6 +53,42 @@ class ClickElement extends BaseElementCommand {
   get elementProtocolAction() {
     return 'clickElement';
   }
+
+
+  getWebElement(webElementOrString) {
+    if (webElementOrString && (webElementOrString.webElement instanceof WebElement)) {
+      webElementOrString = webElementOrString.webElement;
+    }
+
+    if (webElementOrString instanceof WebElement) {
+      return webElementOrString;
+    }
+
+    if (isString(webElementOrString)) {
+      return new WebElement(this.transport.driver, webElementOrString);
+    }
+
+    throw new Error(`Unknown element: ${webElementOrString}`);
+  }
+
+  async executeProtocolAction(actionName, args = []) {
```

Comment:
use `protocolAction` method instead. I think best way to handle this to add a function in method-mapping which takes scriptFn and elementId and runs the script for that elementId as a param

Reply:
Done 👍